report-viewer/src/version.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I remember that we discussed that this file should contain the version number `0.0.0` so that it always shows the development information. The deployed version will have this file automatically replaced in the github actions pipeline.

Reply:
@sebinside Ah, i forgot this. So i just need change this number to 0.0.0, right?